SFA_ADMIN is replaced with STOREFULFILLMENT_ADMIN (#1359)
hotwax/oms-documentation: documents/system-admin/administration/user-permissions/fulfillment-app.md

# Fulfillment App

> HotWax's `Fulfillment App` lets store teams pick and pack orders without special permissions. All users can the app and perform basic tasks. Actions like rejecting orders or marking items as shipped need specific permissions.

Reviewer: ```suggestion
The `Fulfillment App` lets store teams pick and pack orders without special permissions while actions like rejecting orders or marking items as shipped need specific permissions.
```
